### ISSUE TITLE
addLeads with contact

### ISSUE THREAD

**Issue by manylov** [Bolid1/npm-api-client-amocrm#1]
amoClient.listContacts({}) дает
```
[{ id: 3339265,
    name: 'test contact',
    last_modified: 1522687374,
    account_id: 19013968,
    date_create: 1522687374,
    created_user_id: 2287111,
    modified_user_id: 2287111,
    responsible_user_id: 2287111,
    group_id: 0,
    closest_task: 0,
    linked_company_id: null,
    company_name: '',
    tags: [],
    profiles: null,
    type: 'contact',
    custom_fields: [],
    linked_leads_id: [] } ]

```

потом пытаюсь 
amoClient.addLeads([{name: 'test lead', contacts_id:['3339265']}])
равно как 
amoClient.addLeads([{name: 'test lead', contacts_id:[3339265]}])
или 
amoClient.addLeads([{name: 'test lead', contacts_id:3339265}])

сделка создается - но контакта в ней нет

**Comment by Bolid1** [comment]
А права на изменение контактов и сделок присутствуют?

**Comment by manylov** [comment]
Да, я там админ

**Comment by manylov** [comment]
Обратно -   amoClient.addContacts([{name: 'test contact',  leads_id: ["1283825"]}])

контакт создан - к лиду не прицеплен

**Comment by Bolid1** [comment]
Проверил прямыми запросами в amoCRM.
```
POST https://test0000238.amocrm.ru/private/api/v2/json/leads/set/
Content-Type: application/x-www-form-urlencoded

request[leads][add][0][name]=rowTest&request[leads][add][0][contacts_id]=3780695

```
В ответе информация о том, что сделка успешно создана (1290343), но в сделке, действительно, контакт не наблюдается.

Рекомендую вам обратиться в техническую поддержку amoCRM, ибо у них не работают заявленные в документации возможности.

Баг репорт закрываю, т.к. баг относится не к клиенту для amoCRM, а именно к самому amoCRM

**Comment by manylov** [comment]
в нынешней документации на сайте говорится про апи по адресу 
https://new5abb3a9bc689d.amocrm.ru/api/v2/contacts

Возможно private/api/v2/json это нечто устаревшеее?

**Comment by Bolid1** [comment]
Да, действительно, выпустили новую версию API, сломав линкование между сущностями. Добавил адаптацию API клиента в план работ.

**Comment by Bolid1** [comment]
Обновил endpoint-ы в библиотеке, должно помочь с вашей проблемой. Версия с новыми endpoint-ами 0.6.3
